src/vorta/views/archive_tab.py

```diff
@@ -183,46 +183,22 @@ def archiveitem_contextmenu(self, pos: QPoint):
             return  # popup only for selected items
 
         menu = QMenu(self.archiveTable)
-        menu.addAction(
-            get_colored_icon('copy'),
-            self.tr("Copy"),
-            lambda: self.archive_copy(index=index),
-        )
+        menu.addAction(get_colored_icon('copy'), self.tr("Copy"), lambda: self.archive_copy(index=index))
         menu.addSeparator()
 
         # archive actions
-        archive_actions = []
-        archive_actions.append(
-            menu.addAction(
-                self.bRefreshArchive.icon(),
-                self.bRefreshArchive.text(),
-                self.refresh_archive_info,
-            )
-        )
-        archive_actions.append(
-            menu.addAction(
-                self.bMountArchive.icon(),
-                self.bMountArchive.text(),
-                self.bmountarchive_clicked,
-            )
-        )
-        archive_actions.append(menu.addAction(self.bExtract.icon(), self.bExtract.text(), self.extract_action))
-        archive_actions.append(menu.addAction(self.bRename.icon(), self.bRename.text(), self.cell_double_clicked))
-        # deletion possible with one but also multiple archives
-        menu.addAction(self.bDelete.icon(), self.bDelete.text(), self.delete_action)
-
-        if not (self.repoactions_enabled and len(selected_rows) <= 1):
-            for action in archive_actions:
-                action.setEnabled(False)
-
-        # diff action
-        menu.addSeparator()
-        diff_action = QAction(self.bDiff.icon(), self.bDiff.text(), menu)
-        diff_action.triggered.connect(self.diff_action)
-        menu.addAction(diff_action)
```

Comment:
I guess, ideally, we would have the action menu mimic the layout of the button menu, with a separator before 'diff' and moving 'delete' to the end?

Reply:
That would make sense although we should review the current button layout. I initialls designed it so that it distinguishes between single and multi archive options. But maybe there is a better button order.

Reply:
> I initially designed it so that it distinguishes between single and multi archive options

Let's run with that idea. Maybe list them by number of archives required for the action?
- Single archive only actions: 'mount', 'extract', 'rename'
- two archive only actions: 'diff'
- any number archive actions: 'recalculate', 'delete'

Separators between each of these sections, maybe?

Thoughts?

Reply:
Categorizing by impact would also be possible.

1. Recalculate
2. Diff

Those only extract data of vorta.

3. mount
4. extract

Those do not alter the repo but modify the filesystem

5. rename

This is reversible.

6. delete

This deletes data.

Reply:
I can wrap these changes into this PR. Would we like separators between those categories, or just ordered in a list as you outlined?

Same list order for the context menu as well?

Reply:
Which of these two options do you prefer?



> Would we like separators between those categories, or just ordered in a list as you outlined?

I feel like separators or increased spacing would clutter the ui too much.

> Same list order for the context menu as well?

Recognisable patterns repeated throughout the application usually improves the UX.

Reply:
> Which of these two options do you prefer?

Categorizing by impact as you suggest makes logical sense, and I like 'delete' being the last option as that is intuitive.

I will remove the serparators and match the ordering for both button and context menu 👍

Reply:
Okay, the requested changes have been made. The only difference now between the context menu and the buttons is the 'copy' context menu action. Would you like there to be a 'copy' button as well, or leave it as it is?